blog/_posts/2021-10-05-hacktoberfest.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "ROOT joins Hacktoberfest 2021!"
+layout: archive
+author: Vincenzo Eduardo Padulano
+---
+
+The ROOT GitHub repository is open for [Hacktoberfest](https://hacktoberfest.digitalocean.com/) contributions!
+
+## What is it?
+
+Hacktoberfest is a yearly event that encourages participation in open source communities and projects. Would you like to
+help us with some (not so scary) bugs? Or maybe you have some place in our documentation that you think deserves some
```

Comment:
Consider linking to "good first issue": https://github.com/root-project/root/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22